compiler/fuzz/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "candy-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+candy = { path=".." }
+lazy_static = "1.4.0"
+libfuzzer-sys = "0.4"
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
```

Comment:
Could we lift this up to the parent workspace so that the build cache is also shared?